### ISSUE TITLE
Auth Automodeler ORM references ORM

### ISSUE THREAD

**Issue by nanodocumet** [zombor/Auto-Modeler#34]
Is this expected?

https://github.com/zombor/Auto-Modeler/blob/master/classes/auth/automodeler/orm.php#L94

I thought that by having AM there is no need for ORM, however, I see some references to ORM in the auth driver. But in other places it references AM_ORM

https://github.com/zombor/Auto-Modeler/blob/master/classes/auth/automodeler/orm.php#L58

Shouldn't AM be self-contained and not require ORM?

**Comment by zombor** [comment]
Yes, it should be self contained. I've never used tokens with AM auth, and it seems i missed those lines when converting the driver over.

**Comment by nanodocumet** [comment]
Not tested, what about this?

https://gist.github.com/937184
